### PR TITLE
Mobile-Homepage Pull Request

### PR DESCRIPTION
**Pull request by christianonoh**
I developed a headline section with the following features:
- Nav bar
- images
- icons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+  <title>My Portfolio</title>
+</head>
+<body>
+  <main class="wrapper">
```
Comment (Kossi-stack) [inline index.html:15]: The `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <nav>
+      <img src="assets/images/ic_menu.svg" alt="">
+    </nav>
+    <article class="intro">
+      <header>
```
Comment (Kossi-stack) [inline index.html:22]: - [x] kindly use a section tag here
It represents a generic standalone section of the document. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @christianonoh,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [x] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  display: flex;
+  flex-direction: column;
+  background-image: url(assets/images/image_geometry_1.svg);
+  background-repeat: no-repeat;
+  background-size: 111px 74px;
+  background-color: #fff;
+  margin: 0 auto;
+  padding: 20px;
+  align-items: center;
+}
```
Comment (TadesseDev) [inline style.css]: - [Optional] 🛑🛑 Dear @christianonoh, Please notice your design needs more adjustment to make it look like the one on the figma design. Your `button`  needs to take most of the width and be aligned to the center, and the hamburger button needs to be on the right side. 

### Your design 
Screenshot 2023-02-17 200835
### Figma design 
Screenshot 2023-02-17 200924

**Review by TadesseDev — APPROVED** [review]
### Hi @christianonoh 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  I have left optional suggestions. But I strongly believe you should address them.
barly


## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions
### This is only optional because it has not been mentioned to you before.
- [Optional] 🛑🛑 I see you have an empty readme file, This will affect your professionality as a developer. We need to use a professional readme file in all our projects and for that [Microverse provides a Readme template](https://github.com/microverseinc/readme-template) we can customize and use it. I suggest you do the same. 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by christianonoh** [comment]
Thank you!

**Pull request merged**